GenderPayGap.WebUI/Controllers/Report/ReportStartingController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using GenderPayGap.Core;
+using GenderPayGap.Core.Classes;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.WebUI.Helpers;
+using GenderPayGap.WebUI.Models.Report;
+using GenderPayGap.WebUI.Models.ReportStarting;
+using GenderPayGap.WebUI.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GenderPayGap.WebUI.Controllers.Report
+{
+    [Authorize(Roles = LoginRoles.GpgEmployer)]
+    [Route("account/organisations")]
+    public class ReportStartingController: Controller
+    {
+        
+        private readonly IDataRepository dataRepository;
+        private readonly DraftReturnService draftReturnService;
+
+        public ReportStartingController(
+            IDataRepository dataRepository,
+            DraftReturnService draftReturnService)
+        {
+            this.dataRepository = dataRepository;
+            this.draftReturnService = draftReturnService;
+        }
+        
+        #region GET
+        
+        [HttpGet("{encryptedOrganisationId}/reporting-year-{reportingYear}/report/start")]
+        public IActionResult ReportFiguresGet(string encryptedOrganisationId, int reportingYear)
+        {
+             long organisationId = ControllerHelper.DecryptOrganisationIdOrThrow404(encryptedOrganisationId);
+             
+             ControllerHelper.ThrowIfUserAccountRetiredOrEmailNotVerified(User, dataRepository);
+             ControllerHelper.ThrowIfUserDoesNotHavePermissionsForGivenOrganisation(User, dataRepository, organisationId);
+             ControllerHelper.ThrowIfReportingYearIsOutsideOfRange(reportingYear);
+
+             var viewModel = new ReportStartingViewModel();
+
+             DraftReturn draftReturn = draftReturnService.GetDraftReturn(organisationId, reportingYear);

```

Comment:
I don't think this is needed - its only use is to set a particular property on the view model which isn't actually used by the view.